examples/python/src/utils/frontify_queries/create_asset/create_asset.py

```diff
@@ -0,0 +1,82 @@
+import os
+
+from src.services.FrontifyClient import FrontifyClient
+from src.utils.frontify_queries.create_asset.create_asset_executor import (
+    create_asset_executor,
+)
+
+
+def create_asset(
+    client: FrontifyClient,
+    projectid: str,
```

Comment:
```suggestion
    parentid: str,
```
projectid is deprecated and we should switch to parentid